docs/root/start/install.rst

```diff
@@ -45,10 +43,8 @@ using `Get Envoy <https://www.getenvoy.io/>`__.
 
    $ sudo apt update
    $ sudo apt install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
-   $ curl -sL 'https://getenvoy.io/gpg' | sudo apt-key add -
-   $ # verify the key
-   $ apt-key fingerprint 6FF974DB | grep "5270 CEAC"
-   $ sudo add-apt-repository "deb [arch=amd64] https://dl.bintray.com/tetrate/getenvoy-deb $(lsb_release -cs) stable"
+   $ curl -sL 'https://getenvoy.io/gpg' | sudo gpg --dearmor -o /usr/share/keyrings/getenvoy-keyring.gpg
```

Comment:
hmm - im wondering if we *should* verify the key in some way - checksum perhaps

Reply:
We could verify that the keyring file contains the expected key:

gpg --no-default-keyring --keyring /usr/share/keyrings/getenvoy-keyring.gpg -k | grep 5270CEAC57F63EBD9EA9005D0253D0B26FF974DB

Though I'm curious what exactly we're trying to protect against?

Reply:
> Though I'm curious what exactly we're trying to protect against?

supply chain poisoning

i think better include the verification

Reply:
Ah, fair enough. FWIW the previous command wouldn't have protected against that, as it would have been possible to sneak an extra key into the keyring.

I've added a command to check the SHA256 of the resulting keyring. That's simpler than doing anything with the gpg command-line.